prime-router/src/test/kotlin/validation/MarsOtcElrOnboardingValidatorTests.kt

```diff
@@ -0,0 +1,23 @@
+package gov.cdc.prime.router.validation
+
+import assertk.assertThat
+import assertk.assertions.isFalse
+import gov.cdc.prime.router.ActionLogger
+import gov.cdc.prime.router.fhirengine.utils.HL7Reader
+import org.junit.jupiter.api.Test
+
+class MarsOtcElrOnboardingValidatorTests {
+
+    private val validator = MarsOtcElrOnboardingValidator()
+
+    @Test
+    fun `test can detect an invalid RADxMARS message`() {
```

Comment:
Can we throw in a happy path test to show something that does pass the validation.